README.md

```
@@ -0,0 +1,1206 @@
+        <img src="https://img.shields.io/badge/DOI-10.5281/zenodo.14847397-41B5BE?style=flat" alt="Open World Holidays Framework DOI">
+      </a>
+    </td>
+  </tr>
+</table>
+
+## Install
+
+The latest stable version can always be installed or updated via pip:
+
+``` shell
+$ pip install --upgrade holidays
```

markdownlint-cli2 (0.17.2)

Dollar signs used before commands without showing output
null

(MD014, commands-show-output)